addon/services/excel.js

```diff
@@ -57,8 +65,9 @@ export default class ExcelService extends Service {
           if (range.e.c < C) {
             range.e.c = C;
           }
-          let cell = { v: data[R][C] };
-          if (cell.v == null) {
+          let cellValue = data[R][C];
+          let cell = { v: cellValue };
+          if (cellValue.v == null) {
```

Comment:
this change may have caused a break?

```suggestion
          if (cell.v == null) {
```